src/pyhf/infer/mle.py

```diff
@@ -30,10 +30,10 @@ def twice_nll(pars, data, pdf):
         >>> observations = [51, 48]
         >>> data = pyhf.tensorlib.astensor(observations + model.config.auxdata)
         >>> parameters = model.config.suggested_init()  # nominal parameters
-        >>> twice_nll = pyhf.infer.mle.twice_nll(parameters, data, model)
-        >>> twice_nll
-        array([30.77525435])
-        >>> -2 * model.logpdf(parameters, data) == twice_nll
+        >>> nll = pyhf.infer.mle.nll(parameters, data, model)
+        >>> nll
+        array([15.38762717])
+        >>> -1 * model.logpdf(parameters, data) == nll
```

Comment:
do we need "-1" here instead of just " -model.logpdf"

Reply:
Nope. I'll change that.